exercises/armstrong-numbers/lib/example.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:math' show pow;
```

Comment:
This is a good place to introduce lib/module imports.
Add comment explaining this is a selective import which only imports `pow` however this is not required. 
For a novice student selective import might seem too much and might not understand its use and potential but for experienced devs it's probably a known concept.

Reply:
Most students may not think to look for the repository of the track let alone the solution file.

Though they may want to look at out other people solved the problem via the website. I don't see the value in updating the example as it's not a primary resource for a given user.